react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -202,10 +235,20 @@ const styles = theme => {
             bottom: 0,
 
             '&:hover': {
-                backgroundColor: 'rgba(0, 0, 0, .6)',
+                '& .resizable-filmstrip': {
```

Comment:
Optional suggestion: I would move the style into separate file since it have grown significantly. WDYT?

Reply:
Good idea